Changelog

```diff
@@ -42,17 +24,30 @@ Version 3.3, *2020-XX-XX*
       * Added a policy to define the allowed characters for PINs (#2049)
     * WebUI
       * Added the option to filter tokens by tokenrealm (#545)
-      * A QR code is displayed during enrollment of HOTP, TOTP and PUSH tokens which 
-        can be scanned with the authenticator app for quick key import (#1919).
-      * Adding version hashes to WebUI components to avoid working with outdated data (#1871)
-      * Updated bootstrap and JQuery to new versions (#830 #2061 #2064)
-      * Rework policy matching (#1691 #2024 #2038)  
+      * Prior to enrollment of soft tokens, such as HOTP, TOTP and PUSH the user is
+        offered with a QR codes to direct him to the Authenticator App stores (#1919).
+      * Adding version hashes to WebUI components to avoid working with outdated
+        templates (#1871)
+      * Updated bootstrap and JQuery to new versions (#2061)
+      * Rework policy matching (#1691 #2038)
     * Documentation
       * The documentation was restructured and updated (#1967 #1504 #2049).
-      * Started cleaning up pi.cfg templates (#1831)
     * Tools
       * Added a migration script to update the database schema from 2.23.5 to 3.2.2 (#2040)
 
+  Fixes:
+    * Fix unauthorized statistics view (#1238)
+    * Fix API endpoints to avoid redirects (#1999)
+    * Fix a bug which caused an exception during PSKC key file container import (#1915)
+    * Fix url_decode padding before it could cause any issues (#2000)
+    * Initialize rtype in user_object correctly (#2007)
+    * Fix an inconsistency of start_tls with postgres SQL (#2025)
+    * Fix wrong type splitting of questionnaire token (#2026)
+    * Fix a bug which could cause missing audit entries when using the
+      ContainerAudit module (#2029)
+    * Fix a bug which prevented defining an SQL resolver without a password (#2030)
+    * Fix missing "position" argument on event import with pi-manage (#2036)
+
 Version 3.2.2, 2020-01-17
```

Comment:
Some editor was apparently too intelligent, removing blanks from the line endings...